package-lock.json

```diff
@@ -47,7 +47,7 @@
         "javascript-stringify": "^2.1.0",
         "js-cookie": "^3.0.1",
         "js-yaml": "^4.1.0",
```

Comment:
>        "liquidjs": "10.0.0",
         "lodash": "^4.17.21",
         "lodash-es": "^4.17.21",
         "lunr": "^2.3.9",
        "lunr-languages": "^1.9.0",
        "mdast-util-from-markdown": "^1.2.0",
        "mdast-util-to-string": "^3.1.0",
        "morgan": "^1.10.0",
        "msgpack5rpc": "^1.1.0",
        "next": "12.2.4",
        "parse5": "7.1.1",
        "port-used": "^2.0.8",
        "react": "^17.0.2",
        "react-dom": "^17.0.2",
        "react-markdown": "^8.0.3",
        "react-syntax-highlighter": "^15.5.0",
        "rehype-autolink-headings": "^6.1.1",
        "rehype-highlight": "^6.0.0",
        "rehype-raw": "^6.1.1",
        "rehype-slug": "^5.0.1",
        "rehype-stringify": "^9.0.3",
        "remark-code-extra": "^1.0.1",
        "remark-gemoji-to-emoji": "^1.1.0",
        "remark-gfm": "^3.0.1",
 @@ -14022,34 +14022,35 @@
      "resolved": "https://registry.npmjs.org/path-key/-/path-key-4.0.0.tgz",
      "integrity": "sha512-haREypq7xkM7ErfgIyA0z+Bj4AGKlMSdlQE2jvJo6huWD1EdkKYV+G/T4nq0YEF2vgTT8kqMFKo1uHn950r4SQ==",
      "dev": true,
      "engines": {
        "node": ">=12"
      },
      "funding": {
        "url": "https://github.com/sponsors/sindresorhus"
      }
    },
    "node_modules/lint-staged/node_modules/strip-final-newline": {
      "version": "3.0.0",
      "resolved": "https://registry.npmjs.org/strip-final-newline/-/strip-final-newline-3.0.0.tgz",
      "integrity": "sha512-dOESqjYr96iWYylGObzd39EuNTa5VJxyvVAEm5Jnh7KGo75V43Hk1odPQkNDyXNmUR6k+gEiDVXnjB8HJ3crXw==",
      "dev": true,
      "engines": {
        "node": ">=12"
      },
      "funding": {
        "url": "https://github.com/sponsors/sindresorhus"
       }
     },
     "node_modules/liquidjs": {